modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -531,6 +532,25 @@ private CodegenParameter findPathParameterByName(CodegenOperation operation, Str
         return result;
     }
 
+    private void setChildDiscriminatorValue(CodegenModel parent, CodegenModel child) {
```

Comment:
Suggestion: adding a proper docstring to explain what this function does

Reply:
What do you think of : 
```
    /**
     * Add mapping name from the parent as discriminatorValue in the child if it exists in the parent's mapping section.
     * It will not be overridden if the child already have a 'x-discriminator-value' property setted.
     *
     * @param parent  Were we take the mapping from.
     * @param child   The child that will be populated.
     */
 ```
?